src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: $gap;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: $small-screen) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: $middle-screen) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: $large-screen) {
+    grid-template-columns: repeat(4, $card-width);
+  }
```

Comment:
You can do it with changing variable, like in the lesson

Reply:
You are right, but this structure is more understandable for me and I did this task yesterday before our morning lesson